package.json

```diff
@@ -82,11 +82,13 @@
     "preinstall": "bash preinstall.sh",
     "prepare": "bun prepare:husky",
     "prepare:husky": "husky install",
+    "prepare-function-selectors": "bunx tsx ./script/migration/prepare-function-selectors.ts",
     "propose-safe-tx": "bunx tsx ./script/deploy/safe/propose-to-safe.ts",
     "postinstall": "patch-package",
     "remove-from-diamond": "bun script/tasks/cleanUpProdDiamond.ts",
     "test": "forge test --evm-version 'cancun'",
     "test:fix": "npm run lint:fix; npm run format:fix; npm run test",
+    "migrate-allowlist": "bunx tsx ./script/migration/migrate-allowlist.ts",
```

Comment:
does it make sense to add this to package.json? Since this is one-time use only...
(in theory the same could apply for the function selectors command above)

Reply:
yes that’s true. I just wanted to make it as handy as possible for you. We will likely deprecate it with the next update on WhitelistManagerFacet where the migration logic will be removed. wdyt?